contracts/AnteRibbonBalanceTest.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "./interfaces/AnteTest.sol";
+
+interface IRibbonThetaVault {
+    //Returns the asset balance on the vault.
+    function totalBalance() external view returns (uint256);
+    //Returns the vault's total balance, ncluding the amounts locked into a short position
+    function assetBalance() external view returns (uint256);
+}
+
+// Ante Test to check if Ribbon Theta Vault maintains a total balance above or equal to its asset balance
+contract AnteRibbonBalanceTest is AnteTest("Ribbon Theta Vault Balance above or equal to asset value") {
+    // https://etherscan.io/address/0x0FABaF48Bbf864a3947bdd0Ba9d764791a60467A
+    address public constant ribbonThetaVaultAddr = 0x0FABaF48Bbf864a3947bdd0Ba9d764791a60467A;
+
+    IRibbonThetaVault public ribbonThetaVault = IRibbonThetaVault(ribbonThetaVaultAddr);
+
+    constructor () {
+        protocolName = "Ribbon";
+        testedContracts = [ribbonThetaVaultAddr];
+    }
+    
+    function checkTestPasses() public view override returns (bool) {
```

Comment:
Looking at the Ribbon source code it seems like this will always evaluate to true:

```   /**
     * @notice Returns the vault's total balance, including the amounts locked into a short position
     * @return total balance of the vault, including the amounts locked in third party protocols
     */
    function totalBalance() public view returns (uint256) {
        return lockedAmount.add(IERC20(asset).balanceOf(address(this)));
    }

    /**
     * @notice Returns the asset balance on the vault. This balance is freely withdrawable by users.
     */
    function assetBalance() public view returns (uint256) {
        return IERC20(asset).balanceOf(address(this));
    }```